app/meson.build

```diff
@@ -440,3 +440,11 @@ executable('UsocketClientSample', usocket_client_sample_sources,
   dependencies: [asan_dep, mtl, libpthread]
 )
 endif
+
+# v4l2 to IP sample app
```

Comment:
Add `if not is_windows` to skip this app for Windows.

Reply:
> Add `if not is_windows` to skip this app for Windows.

Done.